OpenTrons/opentrons-api
# feat(shared-data, labware-library): Add eppendorf labware definitions

[AUTH-697](https://opentrons.atlassian.net/browse/AUTH-697) and [AUTH-698](https://opentrons.atlassian.net/browse/AUTH-698)



# Overview



Adding new Eppendorf labware definitions with correct module parameters and stacking offsets to shared data and Labware Library. 

All new labware details can be found in [this spreadsheet](https://docs.google.com/spreadsheets/d/1odyozj6Ns-9g_jCI3DxArd3iP6WjKlBCAyoK1khR0rI/edit?usp=sharing).



## Test Plan and Hands on Testing


[New Labware Testing](https://docs.google.com/spreadsheets/d/1r7ldsZj9NxBprDMEges6QFphoLSJyfhqiwT7xg6-7f8/edit?usp=sharing) : This spreadsheet shows all the tests I completed on each of the Eppendorf labware definitions and other new labware definitions. It also includes the LPC offset data for each new labware when placed on a Temperature Module, Magnetic Block, Heater Shaker, and/or Thermocycler Module.

Testing on Flex:
- Measure stacking offset on selected module adapters using calipers and compare it with definitions.
- Run a protocol on a PVT robot with a PVT pipette to test the labware on compatible modules by going to the 4 corner wells.
- Run the Labware Position Check to test the labware dimensions (accepting up to +/- 0.5mm in both the x and y directions due to spring force on the deck slot).
- Test if the gripper can pick up the labware from any compatible module and place it in any slot without dropping or crashing.

Labware Library:
- Ensure that the new Eppendorf labware appears under the Well Plate section and that their labware images are displayed correctly.

1. Eppendorf twin.tec 96 Well Plate 150 µL
2. Eppendorf 96 Deepwell Plate 1000 µL
3. Eppendorf 96 Deepwell Plate 2000 µL
4. Eppendorf twin.tec 384 Well Plate 45 µL


## Changelog



- Added Eppendorf JSON definitions to shared data
- Updated definitions with module parameters and stacking offsets
- Added Eppendorf labwares and their images to Labware Library and Labware section in Opentrons app 
- Added Eppendorf labwares to the lists in `QuickTransferFlow/constants.ts` file: `SINGLE_CHANNEL_COMPATIBLE_LABWARE`, `EIGHT_CHANNEL_COMPATIBLE_LABWARE`, and `NINETY_SIX_CHANNEL_COMPATIBLE_LABWARE`

## Review requests



## Risk assessment




[AUTH-697]: https://opentrons.atlassian.net/browse/AUTH-697?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[AUTH-698]: https://opentrons.atlassian.net/browse/AUTH-698?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
